source/sonata_simulation.rst

```diff
@@ -513,7 +513,7 @@ List of dictionaries to adjust the synaptic strength or other properties of edge
    target                         text       Mandatory    node_set specifying postsynaptic nodes.
    weight                         float      Optional     Multiplier of ``conductance`` used to adjust synaptic strength. The final [NetCon weight](https://nrn.readthedocs.io/en/8.2.6/python/modelspec/programmatic/network/netcon.html#NetCon.weight) in NEURON is computed as ``weight * conductance``, where ``conductance`` is obtained from the ``edges.h5`` file.
    spont_minis                    float      Optional     Synapses affected by this connection_override section will spontaneously trigger with the given rate.
-   synapse_configure              text       Optional     Provide a snippet of hoc code which is to be executed on the synapse objects affected by this connection_override. Use '%s' to indicate where a reference to the synapse object should be filled.
+   synapse_configure              text       Optional     Provide a snippet of HOC code to be executed on the synapse objects affected by this ``connection_override``. Use ``%s`` as a placeholder where a reference to the synapse object should be inserted. You can also override global variables without using ``%s``. For example ``tau_d_NMDA_ProbAMPANMDA_EMS = 2``. However, it is strongly recommended to limit overrides to **one per variable**. These overrides currently **do not** account for: delays, synapse type, order in the file (behavior may depend on the edge file order). Use with caution, as unintended interactions may occur.
```

Comment:
Since it is pointed out that I missed the info here: https://sonata-extension.readthedocs.io/en/latest/sonata_simulation.html#parameters-required-for-modifications
Maybe it is better to link to part of the documentations such that typical NEURON users can know to use that method of manipulating global values for synapse types. Apologies for my ignorance